src/layouts/Images.jsx

```diff
@@ -75,26 +104,23 @@ export default class Images extends Component {
   onImageSelect = async (event) => {
```

Comment:
Remove the `settingsToggle` method from line 71-79, as well as the `updateNewPageName` method